scripts/intfutil

```diff
@@ -149,6 +150,24 @@ def appl_db_port_status_get(appl_db, intf_name, status_type):
         status = ','.join(new_speed_list)
     return status
 
+def state_db_port_status_get(db, intf_name, type):
+    """
+    Get the port status
+    """
+    full_table_id = PORT_STATE_TABLE_PREFIX + intf_name
+    status = db.get(db.STATE_DB, full_table_id, type)
+    if status in [None, ""]:
+        return "N/A"
+    if type == PORT_SPEED and status != "N/A":
+        status = '{}G'.format(status[:-3])
+    elif type in [PORT_ADV_SPEEDS, PORT_RMT_ADV_SPEEDS] and status not in ["N/A", "all"]:
+        speed_list = status.split(',')
+        new_speed_list = []
+        for s in natsorted(speed_list):
+            new_speed_list.append('{}G'.format(s[:-3]))
```

Comment:
This logic will not work for switches that have less than 1G speed (e.g mgmt switches which run sonic) Please consider the use case as well.

Reply:
Done